build-tools/debian-metadata/oss-xamarin.android-targets.install

```diff
@@ -0,0 +1,11 @@
+bin/Release/bin/*					/usr/bin
```

Comment:
Is this wise, placing un-prefixed binaries into `/usr/bin`? The files we generate are:

* cross-arm
* cross-arm64
* cross-x86
* cross-x86_64
* generator
* llc
* opt
* xabuild

@atsushieno already wants to rename `generator` on Linux so something less generic, and everything except `xabuild` just seems like it would conflict with *something*. For example, the [**llvm-3.5** Debian package](https://packages.debian.org/jessie/amd64/llvm-3.5/filelist) installs `/usr/bin/opt-3.5` and `/usr/lib/llvm-3.5/bin/opt`.

If nothing else, installing a `/usr/bin/opt` will confuse things.

I think we originally "envisioned" the Linux install as being in a subtree, e.g.:

* `/opt/lib/mandroid/bin/llc`
* `/opt/lib/mandroid/bin/cross-arm`

Having `/usr/bin/xabuild` seems like a good idea -- though that would likely need to differ from what we currently emit -- but otherwise, I'm not at all certain about this direction, and "polluting" `$PATH` with internal tools.